js/apps/system/_admin/aardvark/APP/react/src/views/views/viewsList/ViewTile.tsx

```diff
@@ -13,7 +14,9 @@ export const ViewTile = ({ view }: { view: SearchViewTypeWithLock }) => {
       boxShadow="md"
       backgroundColor="white"
       cursor="pointer"
-      onClick={() => history.push(`/view/${encodeURIComponent(view.name)}`)}
+      onClick={() => {
+        window.location.href = `${window.location.origin}${window.location.pathname}#view/${encodedName}`;
```

Comment:
Moving away from the router is the best possible solution right now, because history v4.5.2 onwards has bugs with encode/decode, and pre-v4.9.1 has bugs with window.location.hash. So we can't pin history to v4.5.1 without causing issues

This change should not cause any problems, as we are not using router extensively anyway (yet)